README.md

```diff
@@ -158,6 +158,8 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
       > **NOTE**: Make sure your system Python build is `Mach-O 64-bit executable arm64` by using `file -b $(command -v python)`,
       > or if your system is setup with [pyenv](https://github.com/pyenv/pyenv) by using the `file -b $(pyenv which python)` command.
+      >
```

Comment:
Wrong rebase?

Reply:
seems that, any methods to fix it?

Reply:
Just rebase against main.

Reply:
I just did it using the bot but you still need to update your commit message and push again, thanks!